### PR TITLE
fix lighthouse `unsized image` test for home page

### PR DESCRIPTION
**Pull request by ayushthe1**
**Description**

This PR fixes the lighthouse failing `unsized-images` test as seen [here](https://github.com/layer5io/layer5/actions/runs/4206205144/jobs/7299370862#step:6:123)

**Notes for Reviewers**
This PR needs to be checked on Safari browser before merging. I'll wait for the preview_url to generate and then check the changes in safari

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 831e983eb74a50b84f161d293cb9375b60a61fae at: https://63f25ece5be0791513946806--layer5.netlify.app

**Comment by leecalcote** [comment]
From Safari:

Screenshot 2023-02-19 at 4 45 27 PM

File: src/sections/Home/Projects-home/index.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const Projects = () => {
             <Col sm={12} md={6} lg={3}>
               <Link className="project-card" to="/cloud-native-management/meshmap">
                 <div className="project__block__inner">
-                  <StaticImage loading="lazy" src={meshmapLogo} alt="MeshMap Logo" width={50} height={80} />
+                  <StaticImage loading="lazy" src={meshmapLogo} alt="MeshMap Logo" width={50} height={80} imgStyle={{ width: "25px", height: "40px"}} />
```
Comment (leecalcote) [inline src/sections/Home/Projects-home/index.js:47]:
```suggestion
                  <StaticImage loading="lazy" src={meshmapLogo} alt="MeshMap Logo" width={50} height={80} imgStyle={{ width: "35px", height: "56px"}} />
```

**Review by leecalcote — APPROVED** [review]
Thanks for your work on getting this fixed, @ayushthe1 👍

**Pull request merged**